### ISSUE TITLE
Meeting summary view sprint

### ISSUE THREAD

**Issue by ackernaut** [ParabolInc/action#173]
When a team finishes a weekly action meeting, they want a summary view so everybody can see the progress made in the meeting and clearly see who owns the new Projects and Actions.

**The summary view**:

- [x] Congratulates the team for blazing through `x` number of Agenda Items.
- [x] Communicates how many new Projects and Actions were captured.
- [x] Shows which team members are owners of said Projects and Actions.
- [x] Provides a link to return to the team dashboard

** Side-effects **:
- [x] Rename the `Meeting` table to `MeetingHistory`; add meeting start time and end time to record
- [x] Computes a MeetingHistory entry of new actions and project diffs a la #182 
- [x] Deletes completed agenda items, and other clean-up tasks by calling `endMeeting` mutation

**Extra points** for congruency in design for Meeting Summary view and [Summary Email template](https://github.com/ParabolInc/action/issues/155).

**Estimated effort:** 3 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

![meeting-summary-concept-100 2x](https://cloud.githubusercontent.com/assets/307286/18897400/a9ef4e70-84ef-11e6-8581-288d83947423.png)

**Comment by jordanh** [comment]
@ackernaut when you get a sec, it'd be nice to share your new design here.

**Comment by jordanh** [comment]
Added "side-effects" in @ackernaut's comment above.

For convenience, here's a query to delete completed agenda items:

```javascript
r.table('AgendaItem')
  .getAll("B1nQohCn", {index: "teamId"})
  .filter({ "isCompleted": true })
  .delete()
```

**Comment by ackernaut** [comment]
Should it delete only completed Agenda Items, or all Agenda Items?

**Comment by jordanh** [comment]
Hmm. Good question. Probably all agenda items!

**Comment by mattkrick** [comment]
Ah, i suppose I should chime in here with the schema:

```
isComplete: {
      type: GraphQLBoolean,
      description: 'true if the agenda item has been addressed in a meeting (will have a strikethrough or similar)'
    },
    isActive: {
      type: GraphQLBoolean,
      description: 'true until the agenda item has been marked isComplete and the meeting has ended'
    },
```

So the end meeting might do something like:
```
r.table('AgendaItem')
  .getAll("B1nQohCn", {index: "teamId"})
  .filter({ "isActive": true })
  .update({isActive: false, isComplete: false})
```

**Comment by mattkrick** [comment]
Of course we can change the above to whatever we like, that was just my original thought. Plus, since `agendaId` is a FK on the outcome, it may be useful for reporting.

**Comment by jordanh** [comment]
Ah, good point. Perhaps delete isn't the right operation. We probably just need to transition them to a state that can be filtered.

**Comment by ackernaut** [comment]
Yeah, the insights of agenda item content will be helpful IMO showing relationships to team momentum, need to split, etc. So no delete. I was wondering if we only capture agenda items (as data) that make it to the end of a meeting and/or only capture agenda items that results in at least a project/action. This is a design + product question.

**Comment by ackernaut** [comment]
Meaning, if an agenda item is created and removed before the next meeting end, it is really deleted? Or not considered in insights?

**Comment by ackernaut** [comment]
I added a newer concept design to the main description. We can still iterate, of course.

**Comment by jordanh** [comment]
Note: revised @ackernaut's checklist above.

**Comment by jordanh** [comment]
Closed by #387
